### PR TITLE
QOLDEV-545 split SSM-dependent stack params to separate file

### PR DESCRIPTION
**Pull request by ThrawnCA**
Solr servers don't need the values from Parameter Store and don't have the privileges to access them, so put them into a separate file.

### CODE REVIEW

**Review by duttonw — COMMENTED** [review]
is this missing something, i see the swap for ckanweb/batch but nothing about param store removal for solr?

**Comment by ThrawnCA** [comment]
> is this missing something, i see the swap for ckanweb/batch but nothing about param store removal for solr?

Yes, it was missing the part where the attributes were actually moved out of stackparams into a new file :D

Thanks.

File: recipes/ckanparams.rb [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Obtain some stack attributes for the recipes to use
+#
+
+include_recipe "datashades::stackparams"
+
+# Retrieve attributes from SSM Parameter Store
+node.default['datashades']['log_bucket'] = `aws ssm get-parameter --region "#{node['datashades']['region']}" --name "/config/CKAN/#{node['datashades']['version']}/common/s3LogsBucket" --query "Parameter.Value" --output text`.strip
```
Comment (duttonw) [inline recipes/ckanparams.rb:29]: Should be SOLR be pushing its log here also?

**Review by duttonw — CHANGES_REQUESTED** [review]

File: recipes/ckanparams.rb (hunk 1 above)
Comment (ThrawnCA) [reply recipes/ckanparams.rb:29]: You're right. I'll have to update the policy so Solr can access 'common' Parameter Store values.

**Pull request merged**